SRC/NETWORK/hget.c

```diff
@@ -1526,15 +1659,83 @@ void OpenTcpConnection()
         regs.Words.HL = (int)TcpConnectionParameters;
         UnapiCall(codeBlock, TCPIP_TCP_OPEN, &regs, REGS_MAIN, REGS_MAIN);
     }
-    
+
     if(regs.Bytes.A == (byte)ERR_NO_NETWORK) {
         Terminate(strNoNetwork);
     } else if(regs.Bytes.A != 0) {
         sprintf(Buffer, "Unexpected error when opening TCP connection (%i)", regs.Bytes.A);
+		if ((bHTTPS)&&(regs.Bytes.A == ERR_NO_CONN)) //B should have the reason

```

Comment:
`B should have the reason` - it's C, right?

Reply:
Done :)